src/components/MapView/GroundstationLayers/index.tsx

```diff
@@ -0,0 +1,83 @@
+import GeoJSON from 'geojson';
+import React from 'react';
+import { get } from 'lodash';
+import { GeoJSONLayer } from 'react-mapbox-gl';
+import * as MapboxGL from 'mapbox-gl';
+import { LayersMap } from '../../../config/types';
+
+import groundstationDataJson from '../../../data/groundstations/longterm_data.json';
+
+declare module 'geojson' {
+  export const version: string;
+  export const defaults: Object;
+  export function parse(
+    data: Object,
+    propreties: Object,
+    callback?: Function,
+  ): Object;
+}
```

Comment:
This belongs in `custom.d.ts`. But probably better to use [@types/geojson](https://www.npmjs.com/package/@types/geojson) for more complete definitions (I think this was in your other PR?)

Reply:
It doesn't have the parse function sadly... the types are more for the GeoJSON format itself, not the lib, which is a bit confusing...

Reply:
When i try to move it I am getting issues, not sure how I would go about extending the geojson type already in types?